fitbenchmarking/fitbenchmarking
# Default maxdepth in toctree

#### Description of Work

Fixes #558 

Adds set number of index pages to the docs


#### Testing Instructions

1. Check the number of index pages in the docs is consistent 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
